.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## Checklist
+- Code is compiling ✅ ❌
```

Comment:
I like that you thought about this and indeed it's a pre-requisite. I feel that our GH actions cover this so it's not necessary to include in the checklist.

Reply:
Agreed, it was more for external users